lib/fortschritt/meter.rb

```diff
@@ -44,5 +45,12 @@ def total_elapsed_seconds
     def print!
       Fortschritt.printer.print(self)
     end
+
+    def debounce?
+      return true if @updated_at - @printed_at < 0.01 && !completed?
```

Comment:
@fwolfst The only purpose of that check is to never skip printing the last state. Maybe it would have been easier to read like so: 

```ruby
return true unless @updated_at - @printed_at > 0.01 || completed? 
```